readers-frontend/src/components/SearchInterface.astro

```diff
@@ -0,0 +1,47 @@
+---
+import { Icon } from "astro-icon";
+---
+
+<div
+  class="invisible absolute top-0 bottom-0 left-0 right-0 z-50 flex flex-col bg-whiteish opacity-0 transition-opacity duration-500"
+  id="searchInterface"
+>
+  <button
+    id="searchCloseBtn"
+    class="absolute top-0 right-0 w-16 p-2 text-black"
+  >
+    <Icon name="ic:baseline-close" />
+  </button>
+  <div class="mx-auto my-auto p-4">
+    <form id="searchForm" class="flex flex-row items-center">
+      <input
+        id="searchBox"
+        type="text"
+        class="border-b-2 border-black bg-whiteish text-3xl outline-none md:text-5xl lg:text-7xl"
+        placeholder="What are you looking for?"
```

Comment:
The text for the placeholder is not fully visible on the website. Can you make it a bit smaller please? 
<img width="1680" alt="Screenshot 2023-01-30 at 12 11 25 pm" src="https://user-images.githubusercontent.com/58376513/215473518-5daf9946-ebfa-4c75-b3f6-47cfc66f757c.png">

Reply:
What's your screen resolution? Media query works fine on my screen

Reply:
1920 by 1080